### DISCUSSION TITLE
T-Pot Web Portal No Longer Accessible

### DISCUSSION THREAD

**Discussion by TS-02** [telekom-security/tpotce#1057] [Q&A]
On Thursday, I noticed that my T-Pot had stopped updating attacks.

This morning, I tried to access my web portal. I was prompted to enter my credentials, but the page didn't load.

I tried to enter the web portal address again and I haven't been able to gain access.

**Website Error Message**

This site can’t be reached _*MY IP ADDRESS*_ took too long to respond.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_TIMED_OUT

Please could someone advise me on what to do?

**Comment by TS-02** [comment]
🟢 **UPDATE** -> The web portal is accessible again.

Does anyone know why this might be happening?

**Reply by t3chn0m4g3** [reply]
Most likely the your IP was locked by fail2ban. Use `fail2ban-client` to check.

**Reply by TS-02** [reply]
I would try and do this, but I can't even access the machine via **ssh**. What else could I try?

**Comment by sengkoil** [comment]
Maybe you can try run and check the logs...

Command - docker logs kibana 
To look log about kibana docker

**Comment by TS-02** [comment]
Hi @sengkoil,

Where do I find **kibana**? It's not showing in the docker directory.

![image](https://user-images.githubusercontent.com/52854743/161378070-cd8ec7f0-d33c-4083-b325-ba97c8aaffd3.png)
